script/deploy/safe/propose-to-safe.ts

```diff
@@ -70,34 +79,52 @@ const main = defineCommand({
       required: true,
     },
   },
+  /**
+   * Executes the propose-to-safe command
+   * @param args - Command arguments including network, rpcUrl, privateKey, to address, and calldata
+   */
   async run({ args }) {
     const chain = getViemChainForNetworkName(args.network)
 
-    const config: SafeApiKitConfig = {
-      chainId: BigInt(chain.id),
-      txServiceUrl: networks[args.network.toLowerCase()].safeApiUrl,
+    if (!process.env.MONGODB_URI) {
+      throw new Error('MONGODB_URI environment variable is required')
     }
 
-    const safeService = new SafeApiKit(config)
+    const mongoClient = new MongoClient(process.env.MONGODB_URI)
+    const db = mongoClient.db('SAFE')
+    const pendingTransactions = db.collection('pendingTransactions')
 
     const safeAddress = networks[args.network.toLowerCase()].safeAddress
 
     const rpcUrl = args.rpcUrl || chain.rpcUrls.default.http[0]
     const provider = new ethers.JsonRpcProvider(rpcUrl)
     const signer = new ethers.Wallet(args.privateKey, provider)
 
-    const ethAdapter = new EthersAdapter({
```

Comment:
if we decide to stop using EthersAdapter it might be a good idea to use `viem`  as much as possible and eliminate ethers6.
it's not a must-have for me. Feel free to resolve the conversation.

Reply:
Unfortunately I don't think the Safe libraries have good support for viem at the moment. That's the only reason we still have Ethers here.